x-pack/plugins/file_upload/public/importer/validate_file.ts

```diff
@@ -16,8 +16,8 @@ export function getFileExtension(file: File) {
   return '.' + extension;
 }
 
-export function validateFile(file: File, types: string[]) {
-  if (file.size > getMaxBytes()) {
+export function validateFile(file: File, types: string[], isNoLimit?: boolean) {
```

Comment:
Lets remove isNoLimit and instead pass in a parameter called `options`, typed as `options?: { checkSizeLimit?: number}`